Documentation/spelling_wordlist.txt

```diff
@@ -273,6 +275,7 @@ frontend
 fs
 fwmark
 gRPC
+gc
```

Comment:
I didn't see where this is used in the document, is this failing on master somewhere?

Reply:
when I ran the doc tests I was screamed at that this was failing so it was added, but not sure where it came from... so I just updated the commit in my source repo